pyproject.toml

```diff
@@ -86,7 +86,7 @@ exclude = [
     'execution_engine/split_and_sample/sparkdf_data_splitter\.py',  # 4
     'execution_engine/split_and_sample/sqlalchemy_data_sampler\.py',  # 10
     'execution_engine/split_and_sample/sqlalchemy_data_splitter\.py',  # 22
-    'expectations/core/expect_column_', # 214
+    'expectations/core/expect_column_', # 147
     'expectations/core/expect_compound_columns_to_be_unique\.py', # 3
```

Comment:
If it makes things easier, you could replace this with a more specific regex, rather than trying to fix all `214` warnings at once.

Reply:
If it works for you, I would prefer to do this in the next PR, since I expect this inventory to change drastically from this PR to the next.